xmodule/video_block/transcripts_utils.py

```diff
@@ -856,15 +856,18 @@ def available_translations(self, transcripts, verify_assets=None, is_bumper=Fals
         # to clean redundant language codes.
         return list(set(translations))
 
-    def get_default_transcript_language(self, transcripts):
+    def get_default_transcript_language(self, transcripts, dest_lang):
         """
         Returns the default transcript language for this video block.
 
         Args:
```

Comment:
Will you update this comment to describe `dest_lang`

Reply:
`dest_lang` is the language code we are adding to the `video_block` context in https://github.com/edx/frontend-plugins/pull/13/files#diff-a3440af947ec8dc7f08ee883a381419ff3a2aa66ee9ca3bfa2dfcc035fe15d70R59 which references the language that the user selected when translating the unit content. 

<img width="1792" alt="Screenshot 2024-05-01 at 11 56 43" src="https://github.com/openedx/edx-platform/assets/7064472/b4e227ec-b5a6-45d1-9ed1-eea8ecb1665f">

Reply:
Sorry, I should have clarified, that it should be described in the doc string for the function.